automation/python-tests/test_fml.py

```diff
@@ -0,0 +1,112 @@
+from fml import FmlClient, FmlError, InternalError
+import json
+
+
+def test_instantiate_fml_client():
+    FmlClient('./automation/python-tests/resources/test.fml.yml', 'developer')
+
+
+def test_instantiate_fml_client_fails_if_invalid_yml():
```

Comment:
Probably wanna test for an invalid path as well?

Reply:
Yep absolutely